resources/lang/de/Template.properties

```
@@ -227,6 +227,9 @@ couponWrongCustomerType = "Der Gutschein kann nur von Kunden mit einem anderen K
 crossPriceSpecialOffer = ":price"
 crossPriceRRP = "UVP :price"
 ; cross prices
+; variation price
+variationPrice = "ab :price"
```

```suggestion
dynamicVariationPrice = "ab :price"
```
Den Key könnte man etwas eindeutiger benennen...